cli/src/config/templates.toml

```diff
@@ -402,3 +402,16 @@ coalesce(
   "o",
 )
 '''
+
+git_format_patch_email_headers = '''
+  concat(
+    "From " ++ commit_id ++ " Mon Sep 17 00:00:00 2001\n",
+    "From: ",
+    coalesce(author.name(), name_placeholder),
+    " <" ++ coalesce(author.email(), email_placeholder) ++ ">\n",
```

Comment:
nit: maybe we can just use the default `author` output. I don't think we need placeholder for patch output.